### DISCUSSION TITLE
🔴🔴 How can I use deepgram directly in my react app??

### DISCUSSION THREAD

**Discussion by team-deepgram** [deepgram/community#473] [General help] [answered]
Do i need to setup nodejs backend for using deepgram socket??
I want to implement deepgram in frontend only with my Deepgram API key, can you suggest how to achieve that ??
I want to use live(Streaming) feature of deepgram.
A article is also helpful where I can implement deepgram in frontend if possible.

> [Posted by aryanjangid](https://discord.com/channels/1108042150941294664/1179153040381595708/1179153040381595708)

**Comment by jpvajda** [comment, marked as answer]
Answered in Discord; https://discord.com/channels/1108042150941294664/1179153040381595708
